cmd/config.go

```diff
@@ -25,6 +25,8 @@ type RedisConfig struct {
 	MasterName string `yaml:"master_name"`
 	// Redis address list, format: {host1_name:port},{ip:port}
 	Addrs string `yaml:"addrs"`
+	// Redis Sentinel password
+	SentinelPassword string `yaml:"sentinel_password"`
```

Comment:
May be should we add SentinelUser too? :)

Reply:
I am sorry, sure we should.
I made this patch for my own deployment, we don't use username in our sentinel.
And I forgot to add it because thought to make a pull request hit me later :)
Will make another commit right now.